packages/components/input-number/__tests__/input-number.spec.ts

```diff
@@ -41,6 +41,22 @@ describe('InputNumber.vue', () => {
     })
     expect(wrapper.find('input').element.value).toEqual('1')
   })
+  test('set modelValue undefined to form validate', async () => {
+    const wrapper = _mount({
+      template:
+        '<el-input-number :model-value="formData.num" placeholder="input number"/><p>{{formData.num}}</p>',
+      setup() {
+        const formData = ref({
+          num: undefined,
+        })
```

Comment:
I thin we can just use 

```ts
const num = ref(undefined)
```

Reply:
I think it makes no difference, if you think it needs to be modified, I can modify it